assets/src/scripts/_utils.js

```diff
@@ -11,13 +11,14 @@ export const isCsv = (ext) => ext.toLowerCase() === "csv";
 export const isImg = (ext) =>
   ["gif", "jpg", "jpeg", "png", "svg"].includes(ext.toLowerCase());
 
+export const TEXT_TYPES = ["txt", "md", "py", "yaml", "R", "log"];
 /**
  * Confirm if file is .txt
```

Comment:
This code comment is now out of date, as it's no longer just checking for ".txt"

Reply:
Good catch